<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameIDManager</class>
 <widget class="QDialog" name="GameIDManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Game IDs</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>351</width>
     <height>311</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="newGameIDWidget">
    <widget class="QPlainTextEdit" name="directoryText">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>55</y>
       <width>311</width>
       <height>91</height>
      </rect>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="gameIDText">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>311</width>
       <height>31</height>
      </rect>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="lineWrapMode">
      <enum>QPlainTextEdit::NoWrap</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Enter an ID for this game</string>
     </property>
    </widget>
    <widget class="QPushButton" name="selectDirectoryButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>311</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Select Directory</string>
     </property>
    </widget>
    <widget class="QPushButton" name="addGameIDButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>240</y>
       <width>311</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Add New Game ID</string>
     </property>
    </widget>
    <widget class="QPushButton" name="resetDirectoryButton">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>150</y>
       <width>91</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>0</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Game Save Directory</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="existingGameIDWidget">
    <widget class="QComboBox" name="gameIDComboBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>311</width>
       <height>32</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>0</y>
       <width>191</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Select existing Game ID</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="existingDirectoryText">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>311</width>
       <height>91</height>
      </rect>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="existingGameIDText">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>210</y>
       <width>311</width>
       <height>31</height>
      </rect>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="lineWrapMode">
      <enum>QPlainTextEdit::NoWrap</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Game ID</string>
     </property>
    </widget>
    <widget class="QPushButton" name="existingDirectoryButton">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>58</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Select New Directory</string>
     </property>
    </widget>
    <widget class="QPushButton" name="updateGameIDButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>245</y>
       <width>311</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Update Changes</string>
     </property>
    </widget>
    <widget class="QPushButton" name="deleteGameIDButton">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>278</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete From Device</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>66</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Game Directory</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="switchModeButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>221</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Switch to existing Game IDs</string>
   </property>
  </widget>
  <widget class="QLabel" name="activityLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>251</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>You are adding a new gameID</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="extdataRadio">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>50</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>extdata</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="saveRadio">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>30</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>saves</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
